test/conformance/testdata/conformance.txt

```diff
@@ -15,6 +15,7 @@ test/e2e/apimachinery/watch.go: "should be able to start watching from a specifi
 test/e2e/apimachinery/watch.go: "should be able to restart watching from the last resource version observed by the previous watch"
 test/e2e/apimachinery/watch.go: "should observe an object deletion if it stops meeting the requirements of the selector"
 test/e2e/apimachinery/watch.go: "should receive events on concurrent watches in same order"
+test/e2e/apimachinery/table_conversion.go: "should return a 406 for a backend which does not implement metadata"
```

Comment:
Please remove

Reply:
oh, my mistake, still work in progress at that time
now it's ready to review, please take a look.